src/infrastructure/blockchain/ethereum/EthereumOracleAdapter.ts

```diff
@@ -53,13 +63,16 @@ class EthereumOracleAdapter implements OracleGateway {
   public sendResponse(response: Response): Promise<void> {
     this.pendingResponses.push(response);
     return new Promise((resolve, reject) => {
-      this.emitter.on(EthereumOracleAdapter.finishedEventName(response.requestId), (error) => {
-        if (error) {
-          reject(error);
-        } else {
-          resolve();
-        }
-      });
+      this.emitter.on(
+        EthereumOracleAdapter.finishedEventName(response.requestId),
+        error => {
```

Comment:
Oh, you don't have JSLint for sure. It would tell you to change`error` to `(error)` :) . Install JSLint!